tests/dragonfly/cluster_test.py

```diff
@@ -2580,3 +2580,68 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     assert (await StaticSeeder.capture(nodes[1].client)) == start_capture
+
+
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "cache_mode": "true"})
+async def test_cache_cluster_data_migration(df_factory: DflyInstanceFactory, df_seeder_factory):
+    # Check data migration from one node to another with expiration and eviction
+    instances = [
+        df_factory.create(
+            port=next(next_port),
+            admin_port=next(next_port),
+            vmodule="outgoing_slot_migration=2,cluster_family=2,incoming_slot_migration=2,streamer=2",
+            maxmemory="1G" if i == 0 else "3G",
+        )
+        for i in range(2)
+    ]
+
+    df_factory.start_all(instances)
+
+    nodes = [(await create_node_info(instance)) for instance in instances]
+    nodes[0].slots = [(0, 16383)]
+    nodes[1].slots = []
+
+    await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
+
+    logging.debug("Start seeder")
+    await nodes[0].client.execute_command("DEBUG POPULATE 80000 test 10000 RAND SLOTS 0 16383")
+    # cluster_mode set to off to prevent eviction on the target node after migration is finished
+    seeder = df_seeder_factory.create(
+        keys=1000,
+        val_size=10000,
+        stop_on_failure=False,
+        port=nodes[0].instance.port,
+        cluster_mode=False,
+        multi_transaction_probability=0,
+        max_multikey=1,
+    )
+
+    fill_task = asyncio.create_task(seeder.run())
+    # some time to evict data
+    await asyncio.sleep(10)
```

Comment:
how do you know data is evicted in this test?
I dont see stats check, right?
How do you know data is evicted while migrations is in progress?